atomicapp/nulecule/lib.py

```diff
@@ -0,0 +1,64 @@
+from atomicapp.constants import GLOBAL_CONF, DEFAULT_PROVIDER
+from atomicapp.utils import Utils
+from atomicapp.plugin import Plugin
+
+plugin = Plugin()
+plugin.load_plugins()
+
+
+class NuleculeBase(object):
+    def load(self):
+        pass
+
+    def load_config(self, config={}):
+        config = config or {}
+        for param in self.params:
+            value = config.get(self.namespace, {}).get(param['name']) or \
+                config.get(GLOBAL_CONF, {}).get(param['name']) or \
+                param.get('default')
+            if value is None:
+                value = Utils.askFor(param['name'], param)
+            if config.get(self.namespace) is None:
+                config[self.namespace] = {}
+            config[self.namespace][param['name']] = value
+        self.config = config
+
+    def merge_config(self, to_config, from_config):
+        for group, group_vars in from_config.items():
+            to_config[group] = to_config.get(group) or {}
+            if group == GLOBAL_CONF:
+                if self.namespace == GLOBAL_CONF:
```

Comment:
This `if` is a dead code, isn't it?

If `self.namespace` is `GLOBAL_CONF`, use `GLOBAL_CONF` (== `self.namespace`), otherwise, use `self.namespace`...i.e. it's `self.namespace` everytime..